app/scripts/modules/cloudfoundry/src/serverGroup/details/resize/CloudFoundryResizeServerGroupModal.tsx

```diff
@@ -204,8 +207,8 @@ export class CloudFoundryResizeServerGroupModal extends React.Component<
                 <Modal.Body>
                   <Form className="form-horizontal">
                     {this.renderDesired(formik)}
-                    {this.renderQuota(formik, 'diskQuota', diskQuota)}
-                    {this.renderQuota(formik, 'memory', memory)}
+                    {this.renderQuota(formik, 'diskQuota', 'Disk Mb', diskQuota)}
```

Comment:
Picky, but since this is a labeling concern ;) http://www.dewassoc.com/performance/memory/MB_vs_Mb.htm. Can we make this `Disk (MB)`?

Reply:
Yes I can but then we have to change it in our pipeline stage as well. Do you want that included in this change then?